src/pug/includes/head.pug

```diff
@@ -10,8 +10,8 @@ meta(name="description" content="swag opportunities for developers")
 link(rel="icon" type="image/png" href="/assets/img/logo.png")
 link(rel="apple-touch-icon" size="128x128" href="/assets/img/logo.png")
 
-link(rel="stylesheet" href=bustedAssets.css)
-link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
+link(rel="stylesheet" data-href=bustedAssets.css)
+link(rel="stylesheet" data-href="https://cdn.jsdelivr.net/npm/mobius1-selectr@2.4.13/dist/selectr.min.css")
```

Comment:
This entry should be duplicated into a `noscript` tag so the style can load even without JS enabled.